### ISSUE TITLE
Explore Xgboost as a possible decontamination algorithm

### ISSUE THREAD

**Issue by Gabriel-p** [asteca/ASteCA#408]
> XGboost” (Extreme Gradient Boosting), one of the most powerful packages of machine learning

Given that the decontamination process is a simple binary classification problem, it would seem that this algorithm should be applicable. Nonetheless, we only have training data for a single class (field stars), and I'm not sure if this is a limiting factor.

[A Beginner’s guide to XGBoost](
https://towardsdatascience.com/a-beginners-guide-to-xgboost-87f5d4c30ed7)
